### PR TITLE
[setup] Improve README.md'

### PR DESCRIPTION
**Pull request by cuzik**
Hi hi :cow: 

So, in this PR we only create a better `README.md`.

### CODE REVIEW

**Comment by cuzik** [comment]
[[SETUP] Criar um README.md para a api rails (2)](https://trello.com/c/fp95oK2U/58-setup-criar-um-readmemd-para-a-api-rails-2)

File: README.md [hunk 1]
```diff
@@ -1,24 +1,252 @@
...
+
+```sh
+bundle exec rails generate model ModelName column_name1:column_type1 column_name1:column_type1 ... column_name1:column_type1
+```
+
+see [rails generate](https://guides.rubyonrails.org/command_line.html#rails-generate)
+
+Do correct associations.
+
+#### Testnig
```
Comment (rartner) [inline README.md]: Testing?

File: README.md [hunk 2]
```diff
@@ -1,24 +1,252 @@
...
+namespace 'api' do
+  namespace 'v1' do
+    ...
+    resources :yor_controller_name
+    ...
+  end
+end
+```
+
+#### Testnig
```
Comment (rartner) [inline README.md]: Same here

File: README.md (hunk 1 above)
Comment (cuzik) [reply README.md]: ops, sorry

**Pull request merged**